src/dapp/libexec/dapp/dapp-test

```diff
@@ -5,6 +5,8 @@
 ### Options:
 ###   -v, --verbose         trace ouput for failing tests
 ###   -vv                   trace output for all tests including passes
+###   --fuzz-runs=<number>  number of times to run fuzzing tests
+###   --replay=<string>     rerun a particular test case
```

Comment:
What format should the string here have? Could you fit a very shot description in here?

Reply:
When tests are rejected, I output a string showing exactly how to replay it

Reply:
But, long story short the syntax is `(f(uint,uint),0xdeadbeef` where the second argument is the abiencoding of the function arguments

Reply:
Can the suggestion of `=` be removed here, or the parsing fixed for that case?

Reply:
@livnev what do you mean?

Reply:
@asymmetric see [here](https://github.com/dapphub/dapptools/pull/313#issuecomment-570622166)

Reply:
> Can the suggestion of `=` be removed here, or the parsing fixed for that case?

yeah, the `=` can definitely be removed. I just followed the syntax that was already given for `match`. I don't know what the syntax of bash help is supposed to look like

Reply:
so lets remove it